src/qt/splashscreen.h

```diff
@@ -36,8 +36,8 @@ public Q_SLOTS:
     /** Show message and progress */
     void showMessage(const QString &message, int alignment, const QColor &color);
 
-    /** Sets the break action */
-    void setBreakAction(const std::function<void(void)> &action);
+    /** Sets if cancel is currently possible */
```

Comment:
> I dont really care, but also dont think its particularly unclear?

It's unclear to me what the flag is supposed to indicate or why it wouldn't always be true if shutdown is always possible. Maybe it should be called `fast_shutdown_possible` if that's what it actually means. Acked this because the PR is definitely a simplification, but I don't think the code or the reason for preventing the quit option from working after the upgrade is clear. I think better naming could help.